evaluation/utils.py

```diff
@@ -20,3 +28,76 @@ def save_jsonl(data: list[dict], path: Path):
     with open(path, "w", encoding="utf-8") as f:
         for item in data:
             f.write(json.dumps(item) + "\n")
+
+
+def radar_factory(num_vars):
+    """
+    Create a radar chart with `num_vars` Axes.
```

Comment:
My maths years are well behind me, I'm gonna need some explanation here, too much talk of polar transforms and whatnot :rofl: Why is this needed?

Reply:
Edit: this is coming straight from the `matplotlib` docs, which is perfectly fine, but a comment with a reference is sorely needed: https://matplotlib.org/stable/gallery/specialty_plots/radar_chart.html